scylla-server-rust/src/lib.rs

```diff
@@ -0,0 +1,12 @@
+pub mod controller;
+pub mod error;
+pub mod services;
+
+#[allow(clippy::all)]
+#[allow(warnings)]
+pub mod prisma;
+
+pub mod serverdata;
+
+/// The type descriptor of the database passed to the middlelayer through axum state
+pub type Database = std::sync::Arc<prisma::PrismaClient>;
```

Comment:
i would call this prisma, I mean I get calling it database, however its just the prisma client, which weve called just prisma in other applications. not a big deal, i was a little confused when you said we were able to pass around a reference to the database, thats technically not what this is doing but I get why you might call it that

Reply:
is `PrismaDatabase` or something more unique alright?  The namespace gets hella confusing as the library is called `prisma` and the inner type is called `PrismaClient`.

Reply:
eh we can leave it as Database then.